internal/action/config.go

```diff
@@ -80,20 +81,42 @@ func contains(haystack []string, needle string) bool {
 func (s *Action) setConfigValue(ctx context.Context, store, key, value string) error {
 	debug.Log("setting %s to %s for %q", key, value, store)
 
-	if err := s.cfg.Set(store, key, value); err != nil {
+	level, err := s.cfg.SetWithLevel(store, key, value)
+	if err != nil {
 		return fmt.Errorf("failed to set config value %q: %w", key, err)
 	}
 
+	// in case of a non-local config change we don't need to track changes in the store
+	if level != config.Local {
```

Comment:
If we were to add full support for worktree configs this would become wrong, I think.

Both local and worktree configs should be committed to the store, I tink.
But feel free to ignore this, I don't know when/if we will have worktree support.
And even if the gitconfig package has it there is still the question if it makes sense for gopass to expose that feature.

Reply:
This is an easy fix, adding it